src/components/TeamMemberTasks/ReviewButton.jsx

```diff
@@ -186,16 +198,23 @@ const ReviewButton = ({
             : `Are you sure you have completed the review?`}
         </ModalBody>
         <ModalBody>
-          Please add link to related work:
+          Please add link to related work: (required)
           <Input 
           type='text' 
+          required
           value={link}
           onChange={handleLink}
           />
+          {linkError && <div className="text-danger">{linkError}</div>}
         </ModalBody>
         <ModalFooter>
           <Button
             onClick={(e) => {
+              e.preventDefault();
```

Comment:
If this can be put in a seperate class for reusability and maintainability please.